nncf/torch/quantization/quantize.py

```diff
@@ -202,14 +207,37 @@ def send_to_device(tensor):
                                                get_model_device(model))
 
     clone_model = deepcopy(model)
-    compression_ctrl, compressed_model = create_compressed_model(
+    import os
+    original = os.environ.get('ORIGINAL_QUANTIZE', False)
```

Comment:
What is it used for? Where does this variable come from?

Reply:
This is temporary code to launch PTQ tests on CI, I'll remove it, sorry